lib/index.js

```diff
@@ -97,7 +97,9 @@ module.exports = {
         if (this.maxLineLength) {
           parameters.push(`--max-line-length=${this.maxLineLength}`);
         }
-        if (this.ignoreCodes) {
+        if (typeof this.ignoreCodes !== 'undefined'
```

Comment:
Atom's configuration API handles all this verification for us, all we need to check is the length of the array as we will _always_ get an Array from the API.